client/app/bundles/Restaurants/components/Restaurants.jsx

```diff
@@ -1,25 +1,50 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-const Restaurants = ({ restaurants }) => (
-
-  <div>
-    <h3>
-      Hello, hungry you!
-    </h3>
-    <hr />
-      <ul>
-          {restaurants.map(item => <Restaurant key={item.name} rest = {item}/>)}
-      </ul>
-
-  </div>
+import StarRatingComponent from 'react-star-rating-component';
+
+const Restaurants = ({restaurants}) => (
+
+    <div>
+        <h3>
+            Hello, hungry you!
+        </h3>
+        <hr />
+        <ul>
+            {restaurants.map(item => <Restaurant key={item.name} rest={item}/>)}
+        </ul>
+
+    </div>
 );
 
 
 Restaurants.propTypes = {
-  restaurants: PropTypes.array.isRequired,
+    restaurants: PropTypes.array.isRequired,
 };
 
-const Restaurant = (props) => <li>{props.rest.name}</li>
+const Restaurant = (props) => (
```

Comment:
We usually keep one class in a file... 
I know its small here but just as a habit